concrete_datastore/api/v1/filters.py

```diff
@@ -22,6 +20,48 @@
 )
 
 
+def get_filter_field_type(model_class, param):
+    """
+    Return the type of the target field that we want to filter
```

Comment:
"The return type is a String" would be nice as an aditionnal comment

Reply:
Fixed thank you